src/cascadia/TerminalSettingsModel/Profile.idl

```diff
@@ -35,160 +45,57 @@ namespace Microsoft.Terminal.Settings.Model
         Profile();
         Profile(Guid guid);
 
-        Boolean HasName();
-        void ClearName();
-        String Name;
+        OriginTag Origin { get; };
+
+        PROJECTED_SETTING(String, Name);

```

Comment:
this is not the purpose of the projected setting macro ...

this is in a completely different _project layer_ than the projected setting macro ...

Reply:
I think you may be getting this confused with ViewModelHelpers.idl.h's `OBSERVABLE_PROJECTED_SETTING` and `PERMANENT_OBSERVABLE_PROJECTED_SETTING` macros? I created IInheritable.idl.h in TSM to specifically not cross project layers. And the reason that's being done in this PR is to reduce the overhead of having to manually add `<name>OverrideSource` everywhere.

Reply:
I sure was, but why? Because it's the same name. These are not "projected" in any meaningful way, they're just _settings_.

Reply:
Oh. I thought the use of the word "projected" here meant that they are in the projected WinRT object. I guess I could rename the macro to just `SETTING` then.

Reply:
i'd call it `INHERITABLE_SETTING` or `LAYERED_SETTING`